session1/challenge/client/src/js/components/presentational/ModalPortal.js

```diff
@@ -1,4 +1,5 @@
 import React, { Component } from "react";
+import ReactDOM from 'react-dom';
```

Comment:
You can use the `createPortal` that is being imported in the next line.

Reply:
👍